wdl2cwl/tests/cwl_files/align_and_count_multiple_report.cwl

```diff
@@ -205,6 +205,9 @@ steps:
         baseCommand:
           - bash
           - script.bash
+        arguments:
+          - valueFrom: ${if (inputs.counts_txt.length == 0) {throw "counts_txt must
+                contain at least one item.";} else { return "";}}
```

Comment:
Total hack. Doesn't change the command line if the array has at least one item. Throws a javascript exception if the array is empty, effectively avoiding execution of the tool if the input is malformed.